Fixes #1691: Document (and list) which APOC procs start their own transaction (#2098)
neo4j-contrib/neo4j-apoc-procedures: docs/asciidoc/modules/ROOT/pages/transaction/index.adoc

[[transaction]]
= List of procedures with its own transaction

> :description: This chapter describes a list of procedures with its own transaction in the APOC library.

Reviewer: ```suggestion
:description: This chapter describes the list of procedures that start their own transaction in the APOC library.
```
